### ISSUE TITLE
unable to install FLBRP

### ISSUE THREAD

**Issue by Hanqingpeng** [flr/FLBRP#16]
install.packages("FLBRP", repos="http:///flr-project.org/R")
Installing package into ‘C:/~/R/win-library/3.6’
(as ‘lib’ is unspecified)
Warning in install.packages :
  unable to access index for repository http:///flr-project.org/R/src/contrib:
  cannot open URL 'http:///flr-project.org/R/src/contrib/PACKAGES'
Warning in install.packages :
  package ‘FLBRP’ is not available (for R version 3.6.1)
Warning in install.packages :
  unable to access index for repository http:///flr-project.org/R/bin/windows/contrib/3.6:
  cannot open URL 'http:///flr-project.org/R/bin/windows/contrib/3.6/PACKAGES'

**Comment by iagomosqueira** [comment]
I think its is simply that you have one slash too many, try with
```
install.packages("FLBRP", repos="http://flr-project.org/R")
```

**Comment by Hanqingpeng** [comment]
Thanks a lot. A new problem arose when I used this command: install.packages("FLBRP", repos="http://flr-project.org/R").

make[1]: Leaving directory `/cygdrive/c/Users/HQP/AppData/Local/Temp/RtmpGcf7td/R.INSTALL26684f4e569e/FLBRP/src-x64/adolc'
C:\Rtools\mingw_64\bin\nm.exe: 'sublibs': No such file
c:/Rtools/mingw_64/bin/g++ -shared -s -static-libgcc -o FLBRP.dll tmp.def FLBRP.o FLCoreClasses.o init.o adolc.a -LC:/PROGRA~1/R/R-36~1.1/bin/x64 -lR
adolc.a: error adding symbols: Archive has no index; run ranlib to add one
collect2.exe: error: ld returned 1 exit status
no DLL was created
ERROR: compilation failed for package 'FLBRP'
* removing 'C:/~/R/win-library/3.6/FLBRP'
* restoring previous 'C:/~/R/win-library/3.6/FLBRP'
Warning in install.packages :
  installation of package ‘FLBRP’ had non-zero exit status

**Comment by Hanqingpeng** [comment]
I installed it and selected the version that does not require compilation

**Comment by iagomosqueira** [comment]
Yes, new versions are available for R 4.0, not yet for 3.6. R notices the src package is newer and asks to install it, but you need to have [Rtools](https://cran.r-project.org/bin/windows/Rtools/) installed to do that.

 If you can move to R 4.0 maybe do so.
